CPB_AMS_Gizmo_BLE/cpb_ams_gizmo_ble.py

```diff
@@ -0,0 +1,130 @@
+"""
+This example solicits that apple devices that provide notifications connect to it, initiates
+pairing, prints existing notifications and then prints any new ones as they arrive.
+"""
+
+import time
+import displayio
+import terminalio
+from adafruit_gizmo import tft_gizmo
+from adafruit_display_text.label import Label
+from adafruit_display_shapes.rect import Rect
+from adafruit_bitmap_font import bitmap_font
+import adafruit_ble
+from adafruit_ble.advertising.standard import SolicitServicesAdvertisement
+from adafruit_ble_apple_media import AppleMediaService
+from adafruit_circuitplayground import cp
+
+BACKGROUND_COLOR = 0x444444 # Gray
+TEXT_COLOR = 0xFF0000 # Red
+BORDER_COLOR = 0xAAAAAA # Light Gray
+STATUS_COLOR = BORDER_COLOR
+
+# PyLint can't find BLERadio for some reason so special case it here.
+radio = adafruit_ble.BLERadio() # pylint: disable=no-member
```

Comment:
add the radio name here `radio.name = "Now Playing Gizmo"`

Reply:
done